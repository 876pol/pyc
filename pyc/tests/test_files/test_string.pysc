int main() {
    string s = "12" + "34";

    print(s + "\n");

    s += "hello";

    print(s + "\n");

    string sarr[] = {"hello", "world"};
    print(sarr[0] + " " + sarr[1] + "\n");

    sarr[0] = "Hello";
    sarr[1] = "World!";
    print(sarr[0] + " " + sarr[1] + "\n");

    return 0;
}
