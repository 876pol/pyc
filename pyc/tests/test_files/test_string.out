1234
1234hello
hello world
Hello World!
